luigi/contrib/spark.py

```diff
@@ -122,6 +122,10 @@ def jars(self):
         return self._list_config(configuration.get_config().get("spark", "jars", None))
 
     @property
+    def packages(self):
+        return self._list_config(configuration.get_config().get("spark", "packages", None))
```

Comment:
Let's just merge this for now and deal with the config refactoring in a
separate PR

On Thursday, July 2, 2015, Thomas Walwyn <notifications@github.com> wrote:

> In luigi/contrib/spark.py
> <https://github.com/spotify/luigi/pull/1035#discussion_r33754775>:
>
> > @@ -122,6 +122,10 @@ def jars(self):
> >          return self._list_config(configuration.get_config().get("spark", "jars", None))
> >
> >      @property
> > +    def packages(self):
> > +        return self._list_config(configuration.get_config().get("spark", "packages", None))
>
> Are you suggesting I make a new class just for the new parameter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/luigi/pull/1035/files#r33754775>.
>